omigami/spectra_matching/spectra_matching.py

```diff
@@ -40,7 +41,9 @@
 
 
 class SpectraMatching:
-    _PREDICT_ENDPOINT_BASE = "https://app.omigami.com/seldon/seldon/{algorithm}-{ion_mode}/api/v0.1/predictions"
+    _PREDICT_ENDPOINT_BASE = (
+        f"{HOST_NAME}seldon/seldon/" + "{algorithm}-{ion_mode}/api/v0.1/predictions"
```

Comment:
why string concatenation? 😄

Reply:
because otherwise the second and third '{}' blocks will get evaluated as code but we want to leave it like that so we can `.format` it later

Reply:
the first string is already formatted, while the second one is not on purpose. If I joined both this wouldn't work

Reply:
whoops @bernardolk already answered that

Reply:
@carloszanella I understand why you moved things here but I don't think this is correct for the following reasons:
1. Omigami client code shouldn't care about the concept of OMIGAMI_ENV as this is meant only for accessing the production endpoints by *final users*
2. This code is public and we will move integration tests out of this repository, there is a task for this (https://datarevenue.atlassian.net/browse/OMI-456)

I would very strongly support reverting this and leaving changes to _PREDICT_ENDPOINT_BASE being done in test code (which will be moved to core)

Reply:
ok I removed OMIGAMI_ENV as I described in slack